client/my-sites/plugins/main.jsx

```diff
@@ -47,21 +46,20 @@ import { getSelectedSite, getSelectedSiteId, getSelectedSiteSlug } from 'state/u
 import HeaderButton from 'components/header-button';
 import { isEnabled } from 'config';
 
-const PluginsMain = createReactClass( {
-	displayName: 'PluginsMain',
-	mixins: [ URLSearch ],
+export class PluginsMain extends Component {
+	constructor( props ) {
```

Comment:
nit: we don't need a constructor. it is valid to write code that rely upon the [class property transforms](https://babeljs.io/docs/plugins/transform-class-properties/) like this:

```js
export class PluginsMain extends Component {
  state = this.getPluginsState( this.props );
}
```

Reply:
Good shout! :)